app/src/main/kotlin/io/eugenethedev/taigamobile/ui/screens/commontask/CommonTaskScreen.kt

```diff
@@ -122,10 +125,10 @@ fun CommonTaskScreen(
     val isAssignedToMe by viewModel.isAssignedToMe.collectAsState()
     val isWatchedByMe by viewModel.isWatchedByMe.collectAsState()
 
-    fun createEditStatusAction(statusType: StatusType) = EditAction(
+    fun createEditStatusAction(statusType: StatusType) = SimpleEditAction(
         items = statuses.data?.get(statusType).orEmpty(),
-        selectItem = viewModel::editStatus,
-        isResultLoading = editStatusResult.let { (it as? LoadingResult)?.data == statusType }
+        select = viewModel::editStatus,
+        isLoading = editStatusResult.let { (it as? LoadingResult)?.data == statusType }
```

Comment:
- [ ] -> `(editStatusResult as? LoadingResult)?.data == statusType`? Extra 'sugar'

Reply:
Okay, it looks slightly better